views/_includes/script.pug

```diff
@@ -6,6 +6,26 @@ script.
     })
   }
 
+script.
+  var noInfoBox = document.getElementsByClassName('no-info')[0]
+
+  if (noInfoBox) {
+    var yesNoRadios = document.querySelectorAll('div.multiple-choice__item')
+    
+    var noInput = yesNoRadios[1].childNodes[0]
```

Comment:
Wouldn't work if there are 2 on a page, but let's worry about it when it happens.

Reply:
yeah, I worried about that too, but I guess we can cross that bridge when we get there